metabase/metabase
# feat(sdk): Add interactive dashboards to embedding SDK

Closes https://github.com/metabase/metabase/issues/44153

### Description

This adds InteractiveDashboard component to the Embedding SDK.

**UPD:** issue on demo video caused by BE issue, happens for sandboxed users, even in the hosted main app.

### How to verify

1. Use https://github.com/metabase/shoppy/tree/44161-interactive-dashboard with SDK build from this branch.
2. Start Shoppy (`yarn install-and-start`)
3. Login at https://localhost:3004/login
4. Open https://localhost:3004/dash
5. Check demo dashboard, try to drill into questions

### Demo


https://github.com/metabase/metabase/assets/7983744/33ffae64-262c-46c6-b5fe-6cd5b0803ae8


### Checklist
